src/metaopt/core/worker/__init__.py

```diff
@@ -29,14 +29,16 @@ def workon(experiment):
         trial = experiment.reserve_trial(score_handle=producer.algorithm.score)
 
         if trial is None:
+            log.debug("#### Failed to pull a new trial from database.")
+
             log.debug("#### Fetch most recent completed trials and update algorithm.")
             producer.update()
 
             log.debug("#### Poll for experiment termination.")
             if experiment.is_done:
                 break
 
-            log.debug("#### Failed to pull a new trial from database. Producing...")
```

Comment:
Why suppress this log debug message? I think the previous one is better, don't you think?

Reply:
After I changed something in `worker/__init__.py` the error message " Failed to pull a new trial from database." was not appropriate at that line exactly.